File: src/main/java/me/kuku/yuq/service/QQJobService.java
package me.kuku.yuq.service;

import com.IceCreamQAQ.Yu.annotation.AutoBind;
import me.kuku.yuq.entity.QQJobEntity;

import java.util.List;

@AutoBind
public interface QQJobService {
    QQJobEntity findByQQAndType(Long qq, String type);
    List<QQJobEntity> findByQQ(Long qq);
    List<QQJobEntity> findByType(String type);
    void delByQQ(Long qq);
    void save(QQJobEntity qqJobEntity);
}

File: src/main/kotlin/me/kuku/yuq/service/QQJobService.kt
package me.kuku.yuq.service

import com.IceCreamQAQ.Yu.annotation.AutoBind
import me.kuku.yuq.entity.QQJobEntity

@AutoBind
interface QQJobService {
    fun findByQQAndType(qq: Long, type: String): QQJobEntity?
    fun findByQQ(qq: Long): List<QQJobEntity>
    fun findByType(type: String): List<QQJobEntity>
    fun delByQQ(qq: Long)
    fun save(qqJobEntity: QQJobEntity)
}